common/com/twitter/intellij/pants/util/PantsConstants.java

```diff
@@ -25,6 +25,8 @@
   public static final String PANTS_INI = "pants.ini";
   public static final String PANTS_PEX = "pants.pex";
   public static final String PANTS_LIBRARY_NAME = PANTS_PEX;
+  public static final String ABSTRACT_TEST_RUN_CONFIGURATION_CLASS_NAME =
```

Comment:
Since it's specific to Scala, should this constant have SCALA in it somewhere?

Reply:
Good idea!